corehq/apps/geospatial/tests/test_views.py

```diff
@@ -568,3 +570,142 @@ def _sample_geojson_data(name='test-2'):
         "name": name,
     }
     return data
+
+
+@es_test(requires=[case_search_adapter])
+class TestCasesReassignmentView(BaseGeospatialViewClass):
+    urlname = CasesReassignmentView.urlname
+
+    def setUp(self):
+        super().setUp()
+        self.user_a = CommCareUser.create(self.domain, 'User_A', '1234', None, None)
+        self.user_b = CommCareUser.create(self.domain, 'User_B', '1234', None, None)
+
+        self.case_1 = create_case(self.domain, case_id=uuid4().hex, save=True, owner_id=self.user_a.user_id)
+        self.related_case_1 = create_case(
+            self.domain,
+            case_id=uuid4().hex,
+            save=True,
+            owner_id=self.user_a.user_id
+        )
+        self._create_parent_index(self.related_case_1, self.case_1.case_id)
+
+        self.case_2 = create_case(
+            self.domain,
+            case_id=uuid4().hex,
+            save=True,
+            owner_id=self.user_b.user_id
+        )
+        self.related_case_2 = create_case(
+            self.domain,
+            case_id=uuid4().hex,
+            save=True,
+            owner_id=self.user_b.user_id
+        )
+        self._create_parent_index(self.related_case_2, self.case_2.case_id)
+
+        case_search_adapter.bulk_index(
+            [self.case_1, self.case_2, self.related_case_1, self.related_case_2],
+            refresh=True
+        )
+
+        self.client.login(username=self.username, password=self.password)
+
+    def tearDown(self):
+        self.user_a.delete(self.domain, None, None)
+        self.user_b.delete(self.domain, None, None)
+        CommCareCase.objects.hard_delete_cases(self.domain, [
+            self.case_1.case_id, self.case_2.case_id, self.related_case_1.case_id, self.related_case_2.case_id
+        ])
+        super().tearDown()
+
+    def _create_parent_index(self, case, parent_case_id):
+        index = CommCareCaseIndex(
+            case=case,
+            identifier='parent',
+            referenced_id=parent_case_id,
+            referenced_type='parent',
+            relationship_id=CommCareCaseIndex.CHILD
+        )
+        case.track_create(index)
+        case.save(with_tracked_models=True)
+
+    def _refresh_cases(self):
+        self.case_1.refresh_from_db()
```

Comment:
Nit: In `setUp()` you can put all of these cases into a list which you can then simply loop through here as:
```
for case in self.cases:
    case.refresh_from_db()
```

Reply:
Addressed in 2df41b968e0ff06dada9753f574b696729090849